### DISCUSSION TITLE
Does this firmware support SKR mini E3 for Ender 3 V2?

### DISCUSSION THREAD

**Discussion by Gil80** [Jyers/Marlin#1106] [Q&A] [answered]
hi.
I'm looking to upgrading my board. I understand that there's a pinout change for the LCD screen and I need to compile the firmware myself, but will this firmware work with this board?

On another note, does this firmware work with the [landscape LCD mod](https://www.reddit.com/r/ender3v2/comments/ludq0y/update_to_ender_3_v2_stock_lcd_display_front/?utm_medium=android_app&utm_source=share)?

**Comment by smooreace** [comment]
Search is your friend

**Comment by dwery** [comment]
This firmware is a fork of Marlin, it will support whatever was supported at the time of the latest rebase

**Comment by scaramonga** [comment]
But this does not help anyone who has an SKR board?, so the same question is gonna get repeated.  Don't presume to think that 'all' know, because they can edit files in a code editor.  It's confusing, more so, when there are no config files to work from, as regards the SKR variant, which is why I asked exactly the same question the other day.  The Jyers firmware is popular, and a great choice for people with this machine, but it should be more clear as to what it supports, more so for the 'newbies', and not just a 'search is your friend' response, if the question does get asked.  How about 'stating' that in firmware notes?, along with links to guides etc., if there is not going to be a 'base' profile to work from?, like the Creality variants.   It is rather confusing, but I do understand that one can't please all ;)

**Comment by smooreace** [comment]
The same question will never be asked again by anyone that puts forth 2
seconds of their life searching for the answer prior to wasting someone
elses time answering it again.

**Reply by scaramonga** [reply]
One can't please all I guess?, but gone are the days of helping anyone out, without fear of 'wasting anyone's time' ;)

**Comment by smooreace** [comment]
Nope.. there will always be the lazy few that dont care to look. But just
to be abundantly clear.... the question has been answered already, and that
answer came with a rather lengthy conversation that had some value to it.
If you need your hand held then you're out of luck.

**Reply by scaramonga** [reply]
> Nope.. there will always be the lazy few that dont care to look. But just to be abundantly clear.... the question has been answered already, and that answer came with a rather lengthy conversation that had some value to it. If you need your hand held then you're out of luck.

How dare you.  Nothing to do with laziness.

I'm sorry you feel that way, sad, but fair enough, and no one was asking their hand to be held, but it's fine.  A little help goes a long way in this world, or used to?, seems not now, but be happy with your 'I know, they don't' status, if it makes you sleep better at night?

**Reply by scaramonga** [reply]
Sit back and think, what does it matter to you if I, or the OP get a solution to our problems?  Would it not be 'nice' to help those out?  Nope.

Take care **smooreace**, I really hope life deals you well moving forward, I really do.

**Comment by smooreace** [comment]
Don't even pretend to think you know a damned thing about who I am! My
SUCCESS in life came investing in myself rather than expecting others to do
shit for me. In that, I retired at 42 and have nothing better to do but
waste money and time laughing at entitled losers.

Your welcome

**Reply by scaramonga** [reply]
59 here, and still learning, and certainly not 'entitled' to anything ;), but you have told me exactly who you are, although I really wasn't interested in first place, but your demeanour speaks volumes, so yes, you are most welcome.

SKR 2.0 board arriving tomorrow, so there is no point asking for help with that in here also?, as I don't wanna seem an 'entitled' loser, and perhaps mess with your SUCCESS?? ;), WTF!?: lol.

**Reply by Tuncay-Ayhan** [reply]
Smooreace likes to be an a-hole. Never saw him answer anything normal. He reacts to almost any issue, even it he doesn't want to help. He likes to be petted and be told "Good boy".

If I could do it, I would've blocked and banned him from Github.

**Comment by smooreace** [comment]
Funny, your pouting fits your description of me.... exactly. Now run along
and get a tissue.

**Reply by Tuncay-Ayhan** [reply]
> Funny, your pouting fits your description of me.... exactly. Now run along and get a tissue.
> […](#)
> On Sun, Jul 4, 2021 at 4:37 AM Tuncay ***@***.***> wrote: Smooreace likes to be an a-hole. Never saw him answer anything normal. He reacts to almost any issue, even it he doesn't want to help. He likes to be petted and be told "Good boy". If I could do it, I would've blocked and banned him from Github. — You are receiving this because you commented. Reply to this email directly, view it on GitHub <[#1106 (reply in thread)](https://github.com/Jyers/Marlin/discussions/1106#discussioncomment-962202)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AIPE4Q73T4KVO4YI25PYLKLTWATWNANCNFSM47YH3F3Q> .

You must be a very loved person IRL

**Comment by smooreace** [comment]
Facts

Whats hilarious about all the tears and tantrums is that I make a fortune
helping people. A good part of that is because I push people to ask
intelligent, educated questions. I cant even begin to count how many people
come back to me, sometimes years later, and thank me ripping them a new one
for asking a stupid question. Sure... far more never do, and the majority
of those continue to be a leach on society.

But hey... you're absolutely entitled to your opinion! That
doesn't inherently make it right, valid, or even intelligent. But you go
right ahead and have it! Because you're special like that!

**Reply by Tuncay-Ayhan** [reply]
It's not about being intelligent and asking educated questions, it's about how you treat each other.

You have no right to treat someone else like dog-shit. You laugh at them when they don't know the answer. Who are you helping with the words "_I make a fortune helping people_". No one is paying you here. 
They ask a question, they don't want your life lessons. 

I you don't want or can't help someone, just move on and be done with it. I certainly don't know all of the answers to the questions asked here. No one is blaiming you if you can't answer them.

**Comment by smooreace** [comment]
More tears... doing exactly what you're crying about. Eventually you might
get it!

The FACT is that my original response was not only 100% accurate and
correct, but if that advice would have been taken the OP would have
instantly found himself with all the info needed. Not only was it exactly
the info he was searching for, but it also would have instilled a slight
bit of self-worth through his using the tools afforded to him to FIND the
info he wanted, rather than expecting it being handed to him on a silver
platter.

More fact is that DO get paid fairly well assisting people in the 3d
printing realm, though not nearly what I do in other areas. Even more fact
is that in all of your tear-jerking BS of comments here you have offered
NOTHING except that... crying. You're doing precisely what you're trying to
blame me for. Pot... meet kettle!

**Reply by Tuncay-Ayhan** [reply]
You are a funny individual

**Reply by Gil80** [reply]
I actually did search for my question and didn't find a conclusive answer. So how about thinking that my search was poor instead of auto assuming I didn't search?

**Reply by gothcha** [reply]
@Gil80 check out #952 #861 #502  I generally find scrolling back through posts helps, search does not always prove successful for some reason.

**Reply by smooreace** [reply]
works great 
![image](https://user-images.githubusercontent.com/35540547/124498437-f3754280-dd81-11eb-90c0-7219f5381f15.png)

**Comment by Jyers** [comment, marked as answer]
Gonna mark this as answered. Please try not to turn my discussions into allout brawls okay? No one likes that...

**Comment by smooreace** [comment]
Lol
